### PR TITLE
Added example from README

### PR DESCRIPTION
**Pull request by Krupakar-Reddy-S**
Added the example from `pymc` REAME to the [welcome page](https://www.pymc.io/welcome.html) at `pymc.io`.

This is from issue [pymc #7411](https://github.com/pymc-devs/pymc/issues/7411),
previously raised a PR and discussed [here](https://github.com/pymc-devs/pymc/pull/7446).

----
📚 Documentation preview 📚: https://pymcio--98.org.readthedocs.build/

### CODE REVIEW

**Comment by Krupakar-Reddy-S** [comment]
noticed the last column title is not wrapping properly in the build :(

Screenshot 2024-08-07 163105

**Comment by twiecki** [comment]
I like it! CC @OriolAbril

File: welcome.md [hunk 1]
```diff
@@ -19,6 +19,127 @@ Here is what sets it apart:
...
+
+```python
+import pymc as pm
+
+# Taking draws from a normal distribution
+seed = 42
+x_dist = pm.Normal.dist(shape=(100, 3))
+x_data = pm.draw(x_dist, random_seed=seed)
+
+# Independent Variables:
```
Comment (OriolAbril) [inline welcome.md:36]: I would try to minimize code comments to specific annotations that need to go right beside the code itself. For more textual elements like this I would split the code in two and use markdown

**Review by OriolAbril — APPROVED** [review]
Looks great, we removed the other code example because the im browser execution wasn't working well, I like this one

**Comment by Krupakar-Reddy-S** [comment]
@twiecki should i make that change?

**Comment by twiecki** [comment]
> @twiecki should i make that change?

Yes

**Comment by Krupakar-Reddy-S** [comment]
@twiecki is this good?

image

**Comment by twiecki** [comment]
Thanks!

**Comment by Krupakar-Reddy-S** [comment]
@twekei i have not made the chnage, i wanted to know how it was before making it :(

**Comment by twiecki** [comment]
Oh, jumped the gun. Just make a follow-up PR.

**Pull request merged**